### DISCUSSION TITLE
my account suspond from online game until 23 August and reset all achivment !!!

### DISCUSSION THREAD

**Discussion by nastimch** [YimMenu/YimMenu#3440] [General]
my account suspond from online game until 23 August and reset all achivment
is there someone get same ?
i think suspond from yimenu but am using only telport and don't grieve some people so i don't know why suspond itt
![Uploading Untitled.png…]()

**Comment by D-Borkan** [comment]
no, i don't think from Yim 
do you get money  or use recovery options ?

**Reply by nastimch** [reply]
what mean recovery money !
last thing i did 
just sell bunker with teleport and some kill rivals nighclubs that all my be someone report me !
